src/widgets/block.rs

```diff
@@ -36,6 +36,10 @@ pub use title::{Position, Title};
 /// Without left border───
 /// ```
 ///
+/// Styles are applied first to the entire block, then to the borders, and finally to the titles.
+/// If the block is used as a container for another widget, the inner widget can also be styled.
+/// Styles will be inherited by each level unless they are overridden.
```

Comment:
This is the case here too. `overridden` how? By another style on top? no, by another style with reset or actively setting what to remove (`no_…`).